crates/wasm-compose/example/server/config.yml

```diff
@@ -4,4 +4,5 @@ search-paths:
 instantiations:
   $input:
     arguments:
-      example:service/handler@0.1.0: svc
+      # TODO: add package version (but note that `wit-bindgen` 0.13.0 doesn't seem to handle it correctly)
```

Comment:
Was the error relating to this coming from building the components? I think https://github.com/bytecodealliance/cargo-component/pull/164 might be part of the problem, which should be addressed with `0.3.1` of `cargo-component-bindings`.

Reply:
I didn't dig into it; I'll try again with the new release.

Reply:
I've updated to `cargo-component-bindings` 0.4.0 and all is well.